src/main/java/com/future/function/qa/api/sticky_note/StickyNoteAPI.java

```diff
@@ -0,0 +1,55 @@
+package com.future.function.qa.api.sticky_note;
+
+import static com.future.function.qa.util.Path.STICKY_NOTE;
+
+import com.future.function.qa.api.BaseAPI;
+import com.future.function.qa.model.request.sticky_note.StickyNoteWebRequest;
+import io.restassured.http.ContentType;
+import io.restassured.http.Cookie;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import net.thucydides.core.annotations.Step;
+
+public class StickyNoteAPI extends BaseAPI {
+
+  @Step
+  public Response createStickyNote(StickyNoteWebRequest request, Cookie cookie) {
+
+    if (cookie == null) {
+      return createStickyNoteWithoutCookie(request);
```

Comment:
maybe if future tests is going to be like this, can be moved to a new helper class

Reply:
Will change in future PR, maybe in CORE-QA-3